CardEffect/ST22/ST22_06.cs

```diff
@@ -0,0 +1,371 @@
+                    }
+                }
+
+                bool CanSelectPermanentCondition(Permanent permanent)
+                {
+                    if (CardEffectCommons.IsPermanentExistsOnOpponentBattleArea(permanent, card))
+                    {
+                        if (permanent.IsDigimon)
+                        {
+                            if (!permanent.TopCard.CanNotBeAffected)
+                            {
+                                if(CardEffectCommons.IsMinDP(permanent, card.Owner.Enemy));
+                                {
+                                    return true;
+                                }
+                            }
+                        }
+                    }
+                }
+
+                IEnumerator ActivateCoroutine(Hashtable hashtable)
+                {
+                    if (CardEffectCommons.HasMatchConditionPermanent(CanSelectPermanentCondition))
+                    {
+                        Permanent selectedPermanent = null;
+                        int maxCount = Math.Min(1, CardEffectCommons.MatchConditionPermanentCount(CanSelectPermanentCondition));
+                        SelectPermanentEffect selectPermanentEffect = GManager.instance.GetComponent<SelectPermanentEffect>();
+
+                        selectPermanentEffect.SetUp(
+                            selectPlayer: card.Owner,
+                            canTargetCondition: CanSelectPermanentCondition,
+                            canTargetCondition_ByPreSelecetedList: null,
+                            canEndSelectCondition: null,
+                            maxCount: maxCount,
+                            canNoSelect: true,
+                            canEndNotMax: false,
+                            selectPermanentCoroutine: SelectPermanentCoroutine,
+                            afterSelectPermanentCoroutine: null,
+                            mode: SelectPermanentEffect.Mode.Custom,
+                            cardEffect: activateClass);
+
+                        selectPermanentEffect.SetUpCustomMessage("Select 1 Digimon to place in security", "The opponent is selecting 1 Digimon place in security.");
+
+                        yield return ContinuousController.instance.StartCoroutine(selectPermanentEffect.Activate());
+
+                        IEnumerator SelectPermanentCoroutine(Permanent permanent)
+                        {
+                            selectedPermanent = permanent;
+                            yield return null;
+                        }
+
+                        if (selectedPermanent != null)
+                        {
+                            yield return ContinuousController.instance.StartCoroutine(new IDestroySecurity(
+                        player: card.Owner.Enemy,
+                        destroySecurityCount: 1,
+                        cardEffect: activateClass,
+                        fromTop: true).DestroySecurity());
+                        }
```

Comment:
You forgot to add the part of moving selectedPerament to bottom security.

```suggestion
                        if (selectedPermanent != null)
{
    yield return ContinuousController.instance.StartCoroutine(new IPutSecurityPermanent(
        permanent: selectedPermanent,
        hashtable: hashtable,
        toTop: false).PutSecurity());

    yield return ContinuousController.instance.StartCoroutine(new IDestroySecurity(
    player: card.Owner.Enemy,
    destroySecurityCount: 1,
    cardEffect: activateClass,
    fromTop: true).DestroySecurity());
}
```